dist/templates/k8s.ovn.org_clusteruserdefinednetworks.yaml.j2

```diff
@@ -114,9 +114,17 @@ spec:
                           When omitted, the platform will choose a reasonable default which is subject to change over time.
                         items:
                           type: string
+                          x-kubernetes-validations:
+                          - message: CIDR is invalid
+                            rule: isCIDR(self)
                         maxItems: 2
                         minItems: 1
                         type: array
+                        x-kubernetes-validations:
+                        - message: When 2 CIDRs are set, they must be from different
+                            IP families
+                          rule: size(self) != 2 || !isCIDR(self[0]) || !isCIDR(self[1])
```

Comment:
yeah, that is a strange logic of CEL. If the condition of CEL is not true, an error will be returned, but we don't want to fail all conditions e.g. if the CIDR is wrong. So isCIDR will be validated separately by the CIDR struct CEL ans say "wrong CIDR format", but if ^ this CEL fails it will say "When 2 CIDRs are set, they must be from different  IP families", together with the wrong cidr message, which makes it more confusing.
So we only want to fail a given CEL when a specific problem happens (considering other failures are covered by other CELs), in this case we kinda "return true" is the CIDR is wrong, because we just can't check which ip families exist AND we know worng cidr is covered by another CEL